lib/states/wi/madison.R

```diff
@@ -8,19 +8,48 @@ source("common.R")
 # each year, this seems reasonable.
 load_raw <- function(raw_data_dir, n_max) {
   # NOTE: "IBM" is the officers department ID
-  cit <- load_single_file(
+  cit_old <- load_single_file(
     raw_data_dir,
     "mpd_traffic_stop_request_citations.csv",
     n_max
   )
-  warn <- load_single_file(
+  warn_old <- load_single_file(
     raw_data_dir,
     "mpd_traffic_stop_request_warnings.csv",
     n_max
   )
+
+  cit_new <- load_single_file(
+    raw_data_dir,
+    "elcis_with_demographics.csv",
+    n_max
+  )
+
+  warn_new <- load_single_file(
+    raw_data_dir,
+    "warnings_with_demographics.csv",
+    n_max
+  )
+
+  ## NOTE: The new data starts in Jan. 2018 
+  ## and the old data covers through Sept. 2017, 
+  ## so we're still missing Oct.-Dec. 2017
+  new_d <- bind_rows(
+    cit_new$data,
+    warn_new$data
+  ) %>%
+    rename(
+      State = `Reg State`,
+      Year = `Model Year`,
+      Ticketed = Type
+    )
```

Comment:
Just to be consistent with Dan and Amy, I would do the `rename`ing in `clean` after coalescing the corresponding columns.